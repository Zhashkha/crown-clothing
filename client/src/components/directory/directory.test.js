import React from "react";
import { shallow } from "enzyme";

import { Directory } from "./directory.component";
import MenuItem from "../menu-item/menu-item.component";

describe("test Directory", () => {
  let wrapper;
  let mockSections;

  beforeEach(() => {
    mockSections = [{ id: 1 }, { id: 2 }];
    wrapper = shallow(<Directory sections={mockSections} />);
  });

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should render same amout of MenuItem elements as mock sections are", () => {
    expect(wrapper.find(MenuItem).length).toBe(mockSections.length);
  });
});
