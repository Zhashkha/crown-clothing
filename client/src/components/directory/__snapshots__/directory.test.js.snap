// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Directory should match snapshot 1`] = `ShallowWrapper {}`;
